.rubocop.yml

```diff
@@ -1,6 +1,10 @@
 inherit_gem:
   prawn-dev: rubocop.yml
 
+AllCops:
+  Exclude:
+    - vendor/**/*
```

Comment:
I guess this might make sense to have in https://github.com/prawnpdf/prawn-dev/blob/main/rubocop.yml, if you agree could you commit that?

It's also possible to use
```yml
inherit_mode:
  merge:
    - Exclude
```
(https://docs.rubocop.org/rubocop/0.88/configuration.html#merging-arrays-using-inherit_mode)

since `vendor/**/*` is [excluded by default](https://github.com/rubocop-hq/rubocop/blob/e12d1eadea402f955aa9be16ff6bd7def6a970d8/config/default.yml#L66) in rubocop.

Reply:
This is why I had the set up in the first place. I imagine Rubocop is not the only tool that can be tripped by vendored gems.

Reply:
RuboCop actually handles it by default, it just unfortunately doesn't default to merge Excludes yet.
It seems very sensible for any tool to ignore `vendor/`, what better place is there for "vendored code"?

Reply:
`bundle install --deployment` also uses that path, and it's frequent to see `bundle config --local path vendor/bundle`, so I think that's valuable to support, even without considering CI, as some devs might do that anyway (e.g., I do).

Reply:
I'm still not keen on changing the source tree. Any chance this path could be configured in the action? Would it be valuable to formally request this feature on the project page?

Reply:
Yes, please open an issue on https://github.com/ruby/setup-ruby/issues about that.
`vendor/bundle` is the typical and most used Bundler path, but I can see it causes issues in some cases, so something outside the working dir also has advantages.

Reply:
FWIW TravisCI also uses `vendor/bundle` by default, but it seemed to be somewhat configurable: https://docs.travis-ci.com/user/caching/#bundler